.github/workflows/tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Tests
+
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Setup Hugo
+        uses: peaceiris/actions-hugo@v2
+        with:
+          hugo-version: "latest"
+          extended: true
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v1
+        with:
+          start: npm start
+          record: true
+          wait-on: http://localhost:1313
+        env:
+          # pass the Dashboard record key as an environment variable
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
```

Comment:
I'll need to configure this in the repo settinngs. Will need permission here @erickhun

Reply:
you should be admin now @geekyme